allennlp/common/plugins.py

```diff
@@ -56,13 +55,20 @@ def discover_plugins() -> Iterable[str]:
     """
     Returns an iterable of the plugins found.
     """
+    plugins: Set[str] = set()
     if os.path.isfile(LOCAL_PLUGINS_FILENAME):
         with push_python_path("."):
```

Comment:
The global one doesn't need this?

Reply:
I guess the assumption made with global plugins is that they should be installed in your virtual environment, rather than just existing in the current directly.